src/main/java/core/basesyntax/strategy/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.FruitDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.